### ISSUE TITLE
Button componet errors

### ISSUE THREAD

**Issue by builat** [netdata/netdata-ui#25]
* Component cannot be overrided. 
```typescript
// For example
// this does not works
const MyBtn = styled(Button)`
  font-size: 200px;
`
```
* If I will render button without additional props I will have:
![image](https://user-images.githubusercontent.com/3184538/68307435-d1737a80-00dd-11ea-9234-6edbdad16992.png)

Also, not a bug, but not usefull for me. I need to import `enum` of button types instead of writing it as simple string.

**Comment by VLegakis** [comment]
Thanks Dmitry. Let me think a bot about this. So you say you want to override the styles of buttons? This may be possible but isn't it against the reason we create the UIKit so we have consistent styling? Maybe for your case I could create another button type which has the styles you want?

**Comment by builat** [comment]
@VLegakis I just want to override some part of styles not a lot of them. So you will never can create all the variations of button, so more rational to give user posibility of overriding.

Also if we talking about "SKIP" `borderless` does not works either.

**Comment by builat** [comment]
Closed via PR: #27
